runner.yml.dist

```diff
@@ -8,7 +8,7 @@ drupal:
     user: "root"
     password: ""
   post_install:
-    - "./vendor/bin/drush en oe_core oe_theme_test -y"
+    - "./vendor/bin/drush en oe_theme_test oe_theme_helper -y"
```

Comment:
I wouldn't always enable oe_theme_test. I understand that for development phase it's handy, but still I would keep it out.

Reply:
Actually about this: we should consider patching core with [this patch](https://www.drupal.org/project/drupal/issues/474684#comment-12419246) so that we can just list the theme helper module as a dependency of the theme and having it enabled when enabling the theme itself (i.e. we could then remove the change above).

Reply:
Ok We will address them and check if the patch can be applied.

Reply:
Patch tested and validated: https://www.drupal.org/project/drupal/issues/474684#comment-12470938